website/cue/reference/releases/0.43.0.cue

```diff
@@ -0,0 +1,449 @@
+
+### Enhancements
+
+- `decode_punycode` and `encode_punycode` with `validate` flag set to false should be faster now, in cases when input data needs no encoding or decoding. (https://github.com/vectordotdev/vrl/pull/1104)
+- `vrl::value::Value` now implements `PartialCmp` that first checks whether the enum discriminants
+	(that both are floats for example), and if they are calls `partial_cmp` on the inner values.
+	Otherwise, it will return `None`. (https://github.com/vectordotdev/vrl/pull/1117)
+- The `encode_proto` function was enhanced to automatically convert valid string fields to numeric proto
+	fields. (https://github.com/vectordotdev/vrl/pull/1114)
+
+### Fixes
+
+- The `parse_groks` VRL function and Datadog grok parsing now catch the panic coming from `rust-onig` on too many regex match retries, and handles it as a custom error. (https://github.com/vectordotdev/vrl/pull/1079)
+- `encode_punycode` with `validate` flag set to false should be more consistent with `validate` set to true, turning all uppercase character to lowercase besides doing punycode encoding (https://github.com/vectordotdev/vrl/pull/1115)
+- Removed false warning when using `set_semantic_meaning`. (https://github.com/vectordotdev/vrl/pull/1148)
+"""
+	  },
+		{
+			type: "feat"
+			description: """
+				The Elasticsearch sink can now write to Amazon OpenSearch Serverless.
+				"""
+			contributors: ["handlerbot", "AvihaiSam"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				Add ability to encode messages to CEF format with the `cef` encoder (widely used in SIEM systems).
+				"""
+			contributors: ["nabokihms"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				The Kubernetes Logs source can now enrich logs with pod information on Windows.
+				"""
+			contributors: ["damoxc"]
+		},
+		{
+			type: "fix"
+			description: """
+				`exec` and `http_server` sources no longer attach a redundant `timestamp` field
+				when log namespacing is enabled.
+				"""
+			contributors: ["rwakulszowa"]
+		},
+		{
+			type: "feat"
+			description: """
+				Allows for chunked gelf decoding in message-based sources, such as UDP sockets or unix datagram sockets. Implementation is based on [Graylog's documentation](https://go2docs.graylog.org/5-0/getting_in_log_data/gelf.html#GELFviaUDP).
+
+				This framing method can be configured via the `framing.method = "chunked_gelf"` option in the source configuration.
+				"""
+			contributors: ["jorgehermo9"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				The `sample` transform can now take in a `group_by` configuration option that will allow logs with unique values for the patterns passed in to be sampled independently. This can reduce the complexity of the topology, since users would no longer need to create separate samplers with similar configuration to handle different log streams.
```

Comment:
```suggestion
				The `sample` transform can now take in a `group_by` configuration option that allows logs with unique values for the patterns passed in to be sampled independently. This can reduce the complexity of the topology, since users no longer need to create separate samplers with similar configurations to handle different log streams.
```